packages/openactive-integration-tests/test/features/access/access-pass-barcode-broker-provided/implemented/access-barcode-broker-test.js

```diff
@@ -14,10 +13,9 @@ FeatureHelper.describeFeature(module, {
   testOpportunityCriteria: 'TestOpportunityOfflineBookable',
   // The secondary opportunity criteria to use for multiple OrderItem tests
   controlOpportunityCriteria: 'TestOpportunityBookable',
+  skipBookingFlows: ['OpenBookingApprovalFlow'],
```

Comment:
I've put this here to explicitly state that Approval has been tried and doesn't work on this test